### PR TITLE
Add EIP: Standardize Ethereum Drink Menu

### PR DESCRIPTION
**Pull request by SamWilsn**

### CODE REVIEW

**Comment by SamWilsn** [comment]
Moved to draft to prevent automerge, it's ready once an editor reviews and approves.

**Comment by MicahZoltu** [comment]
I !@#$ing hate informational EIPs.

**Review by Bezzat — APPROVED** [review]

**Comment by SamWilsn** [comment]
As far as I can tell I pass the bot? I'm just worried it'll automerge if I take it out of draft... Let's see though!

**Comment by MicahZoltu** [comment]
You pass CI, but not the bot: https://github.com/ethereum/EIPs/pull/4959#issuecomment-1084356066

File: EIPS/eip-4959.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+
+##### Ingredients
+
+ * **`250 ml`** water
+ * **`250 ml`** sugar
+ * **`25 ml`** Earl Grey tea leaves
+
+##### Preparation
+
+1. In a saucepan, bring water just to a boil and remove from heat.
```
Comment (Pandapip1) [inline EIPS/eip-4959.md:53]: The numbered lists are all typod.

File: EIPS/eip-4959.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+
+The drinks specified in this EIP use standard ingredients available in most venues.
+
+In the interest of international applicability, this EIP uses metric units over imperial measurements, which is different from (but compatible with) standard North American cocktail recipes.
+
+## Security Considerations
+
+There are negative health effects related to consuming alcohol, and standardizing a drink menu implies a certain level of support for doing so. By finalizing this EIP, the community confirms this support. It should be noted that this EIP _only_ exists to standardize a drink menu for those who would already be consuming alcohol, and does not aim to increase alcohol consumption at events.
+
+In addition to the above longer term health effects, alcohol has an immediate effect causing impairment. No development or research on the Ethereum protocol or related standards should be performed while under the influence of an excess of these beverages.
```
Comment (Pandapip1) [inline EIPS/eip-4959.md:321]: Should these effects be included in a test cases section?

**Review by Pandapip1 — COMMENTED** [review]
This looks fine.

File: EIPS/eip-4959.md (hunk 1 above)
Comment (SamWilsn) [reply EIPS/eip-4959.md:53]: Markdown automatically numbers the lists when rendered, and this style lets you add/remove items without manually renumbering.

File: EIPS/eip-4959.md (hunk 2 above)
Comment (SamWilsn) [reply EIPS/eip-4959.md:321]: Hm, I think the tests in Test Cases verify that the EIP has been implemented correctly, so for this proposal they'd be taste tests?

File: EIPS/eip-4959.md (hunk 1 above)
Comment (Pandapip1) [reply EIPS/eip-4959.md:53]: Ah, then maybe replace the ones that were manually numbered 2, 3, etc?

File: EIPS/eip-4959.md (hunk 2 above)
Comment (Pandapip1) [reply EIPS/eip-4959.md:321]: Also, cases seem to be a bit too much to test. How about we only need to test 50mL?

**Comment by gcolvin** [comment]
This is absolutely beautiful!

I do not share @MicahZoltu's abhorrence of Informational EIPs, so I would be very tempted to merge this. However, it does need additional information.  Given that many Etherean's need or want to avoid alcohol, but nonetheless want to participate in the festivities, there needs some non-alcoholic versions and/or tasty alternatives.

**Comment by SamWilsn** [comment]
> This is absolutely beautiful!

Thank you!

> I do not share @MicahZoltu's abhorrence of Informational EIPs, so I would be very tempted to merge this. However, it does need additional information. Given that many Etherean's need or want to avoid alcohol, but nonetheless want to participate in the festivities, there needs some non-alcoholic versions and/or tasty alternatives.

I'd be happy to add some! In the mean time, are there any formatting issues that need to be resolved before merging in draft state?

**Comment by MicahZoltu** [comment]
I'm closing all PRs that were open on April 1st since we received a number of "joke" EIPs.  If this was in fact a serious EIP that you would like to have reviewed, please resubmit.

Note: Please keep in mind that EIP editors are all volunteers and have more work than time available.

**Comment by SamWilsn** [comment]
I wouldn't be opposed to getting this merged (and there is [precedent](https://datatracker.ietf.org/doc/html/rfc2324) in other standards organizations), but I'm not exactly going to push very hard to get a joke EIP accepted as a standard :rofl:

**Comment by gcolvin** [comment]
I tend to agree, @SamWilsn, but @MicahZoltu is right that can't do it the IETF way.  I think we should at least make it easy to find these some.  An ACD meeting on April 1 won't happen again for a long, long time ... I started a discussion here:

https://discord.com/channels/595666850260713488/746566142700814426/959664327168786454

**Review by Pandapip1 — COMMENTED** [review]
This is a beautiful EIP.

Would someone else mind ensuring that these recipes are okay? I'm not able to legally test them out. Especially in one go.

**Pull request closed without merging**